main.cpp

```diff
@@ -148,29 +183,61 @@ struct Museum
         int age = 0;
         float satisfaction = 0.5f;
 
-        float payAdmissionFee(float feeAmount) { return feeAmount; }
+        Visitor()
+        {
+            std::cout << "a Museum::Visitor named '" << name << "' was created" << std::endl; 
+        }
+
+        float payAdmissionFee(float feeAmount)
+        {
+            ++timesVisited; // bit confused why this doesn't appear to update the timesVisited int
+            std::cout   << "Museum::Vistor:payAdmissionFee() - A visitor named '" << name << "' payed an admmission fee of $"
+                        << std::fixed << std::setprecision(2) << feeAmount << " and has now visited " << timesVisited << " times" << std::endl;
+            return feeAmount;
```

Comment:
call `std::setprecision(2)` at the top of `main()` so all `std::cout` code that follows will use this precision setting.
- [ ] revise